skbio/alignment/_alignment.py

```diff
@@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
                                           "or fn is provided.")
 
         if ids is not None:
-            fn = lambda _: ids
```

Comment:
> However, from the PEP8 example, maybe we can keep those super simple functions in one line

I'm going to merge this so we can get things passing again. Using one line functions in the future sounds like a good idea but I don't think it's too critical at this point. If someone's available to work on this in a separate PR that'd be great!